tools/clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -7970,6 +7970,14 @@ def err_hlsl_unsupported_semantic_index: Error<
   "'%0' is defined with semantic index %1, but only values 0 through %2 are supported">;
 // HLSL Change Ends
 
+// SER Change Starts
+// Shader Execution Reordering
```

Comment:
```suggestion
// HLSL Change Starts - Shader Execution Reordering
```

Alternatively you could remove the HLSL Change Ends comment above.

Reply:
Removed the HLSL Change end marker in latest version